### DISCUSSION TITLE
Older branches merging?

### DISCUSSION THREAD

**Discussion by practicalrs** [bevyengine/bevy#16391] [Q&A] [answered]
Hi,

I noticed that there are changes on release-0.14.2 branch that look like not merged into release-0.15.0 branch https://github.com/bevyengine/bevy/compare/release-0.15.0...release-0.14.2

I'm not sure about that, maybe GitHub diff shows it wrongly. But usually, projects merge fixes that were made in post-release branches to main to avoid losing these fixes in the new releases. Is it normal practice for Bevy? Is it documented somewhere?

Best regards,
Michal

**Comment by alice-i-cecile** [comment, marked as answer]
From time-to-time we need to backport a fix that can't be merged cleanly due to incompatible changes that have since been merged. Those are the commits you're seeing.
